src/main/java/seedu/address/model/event/EventManpowerNeeded.java

```diff
@@ -8,7 +8,7 @@
  * Represents the number of manpower needed for an Event.
  */
 public class EventManpowerNeeded {
-    public static final String MESSAGE_CONSTRAINTS = "Manpower Needed should be an positive integer between 1 to 999";
+    public static final String MESSAGE_CONSTRAINTS = "Manpower Needed should be an positive integer between 1 to 100";
```

Comment:
100? so small liao?

Reply:
Our target user is a small events company - so there is hardly a possibility of them needing more than 100 employees for a single event.